package.json

```diff
@@ -41,6 +41,7 @@
     "@angular/router": "5.0.2",
     "@types/hammerjs": "2.0.35",
     "core-js": "2.4.1",
+    "font-awesome": "4.7.0",
```

Comment:
Can't we use https://material.io/icons/ ?

Reply:
The material icon set doesn't include any icons for social providers, such as linkedIn and Github, which I'm using on the team member cards

Reply:
Ahha ok thanks 👍

Reply:
Both of these work it's not able to resolve the fonts which is why it's failing.
`@import '~font-awesome/css/font-awesome.min.css';`
`@import './../node_modules/font-awesome/css/font-awesome.min.css';`

Reply:
Also just for social icons, we are importing an entire library. Is there a subset that we can utilize ?

Reply:
@ashwin-sureshkumar it looks like only the woff2 file is being pulled in anyway, so maybe it isn't a major issue for now. Could it possibly be fixed/replaced in another pr? Open an issue for looking into it?

![image](https://user-images.githubusercontent.com/5526934/33964995-aa0b8834-e05a-11e7-893d-3be4dcadf4d9.png)

Reply:
@fredrik-lundin I agree let's deal with that in later PR

Reply:
Sounds good. If there is a bit more research needed

Reply:
Worth looking at: https://material.angular.io/components/icon/overview#font-icons-with-css

Reply:
I agree, this is an awfully big file to pull in for a single icon. We may need to look further at this before merging.

Reply:
@btroncone , @sumitarora , @ashwin-sureshkumar please have a look at my latest commits. I removed FA and replaced it with a custom subset generated by https://icomoon.io/app. Much smaller footprint 👍